examples/custom_logger.js

```diff
@@ -0,0 +1,60 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+const Pulsar = require('../index.js');
```

Comment:
```suggestion
const Pulsar = require('pulsar-client');
```

Reply:
After thinking for a while, I think it's better to use `require('../')` like all dependencies did. See examples of these dependencies.

```
node_modules/minimist/example
node_modules/shoe/example
node_modules/object-inspect/example
node_modules/text-table/example
node_modules/resolve/example
node_modules/concat-map/example
node_modules/xmlhttprequest/example
node_modules/deep-is/example
node_modules/json-stable-stringify-without-jsonify/example
node_modules/engine.io/node_modules/debug/example
node_modules/fast-json-stable-stringify/example
node_modules/json-stable-stringify/example
node_modules/engine.io-client/node_modules/debug/example
```

Reply:
Yes. And it's better to let all examples use `require('../')`